module/solutions/helper.js

```diff
@@ -59,4 +60,83 @@ module.exports = class SolutionsHelper {
     });
   }
 
+  /**
+  * Update User District and Organisation In Solutions For Reporting.
+  * @method
+  * @name _addReportInformationInSolution 
+  * @param {String} solutionId - solution id.
+  * @param {Object} userProfile - user profile details
+  * @returns {Object} Solution information.
+*/
+
+  static addReportInformationInSolution(solutionId,userProfile) {
```

Comment:
@priyanka-TL , @VISHNUDAS-tunerlabs All the comments for survey service are applicable here.